### PR TITLE
Update DIA docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR is an extension of #766; it just makes some general updates to adhere to our standards.

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If this page requires a disclaimer, I have added one

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/integrations/oracles/dia.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+
+1. Access your randomness oracle smart contract on Moonbeam
+2. Call `getLastRound()`to obtain the ID of the latest published round. You can use the **Read Contract** functionality under the **Contract** tab of the contract on Moonscan to execute this call
+3. Call `getRandomValueFromRound(uint256 _round)` using the obtained round ID. Again, you can use Moonscan to quickly execute this call
+
+The response contains the randomness value.
+
+The signature can also be requested by calling `getRandomValueFromRoundWithSignature(uint256 _round)`, which returns a tuple containing the randomness value, the signature, and the previous signature.
+
+To learn how to deploy a randomness-consuming contract on Moonbeam, please refer to the [Deploying a #Randomness Consuming Smart Contract on #EVM chains with DIA xRandom #Oracle](https://youtu.be/BzN-tBgW-xs){target=_blank} video tutorial.
```
Comment (themacexpert) [inline builders/integrations/oracles/dia.md:279]:
```suggestion
To learn how to deploy a randomness-consuming contract on Moonbeam, please refer to the [Deploying a Randomness Consuming Smart Contract on EVM chains with DIA xRandom Oracle](https://youtu.be/BzN-tBgW-xs){target=_blank} video tutorial.
```

Comment (themacexpert) [reply builders/integrations/oracles/dia.md:279]: Not a huge fan of the hashtags in the title

File: builders/integrations/oracles/dia.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+# Learn More About DIA {: #learn-more-about-dia }
+
+- [Twitter](https://twitter.com/DIAdata_org){target=_blank}
+- [Discord](https://discord.gg/dia-dao){target=_blank}
+- [Website](https://diadata.org/){target=_blank}
+- [Docs](https://docs.diadata.org/){target=_blank}
+- [Explore data](https://www.diadata.org/app/){target=_blank}
+
```
Comment (themacexpert) [inline builders/integrations/oracles/dia.md:288]: Screenshot 2023-10-09 at 5 08 00 PM

This last section I would rename Resources and drop it down from a # title to a ## title. Or perhaps keep the name but add a  sentence or two of description. Right now it just feels a bit empty.

**Review by themacexpert — APPROVED** [review]
LGTM 👍

File: builders/integrations/oracles/dia.md (hunk 1 above)
Comment (eshaben) [reply builders/integrations/oracles/dia.md:279]: Totally fair. I just copied it from the actual title on YouTube, but it's not a big deal, so we can remove them!

File: builders/integrations/oracles/dia.md (hunk 2 above)
Comment (eshaben) [reply builders/integrations/oracles/dia.md:288]: Oh yeah didn't realize it was a # title... we don't use those besides the actual title of the page. good catch!

**Pull request merged**